lld/test/wasm/lto/libcall-archive.ll

```diff
@@ -2,7 +2,7 @@
 ; RUN: llvm-as -o %t.o %s
 ; RUN: llvm-as -o %t2.o %S/Inputs/libcall-archive.ll
 ; RUN: llvm-ar rcs %t.a %t2.o
-; RUN: wasm-ld -o %t %t.o %t.a
+; RUN: wasm-ld -mllvm -mattr=-bulk-memory -o %t %t.o %t.a
```

Comment:
Why is this needed?  Should this come from the object file if it needs it?

Reply:
This is an LTO test, so there is no `clang` or `llvm-mc` to pass `-mattr=` to. I guess we could put `-bulk-memory` in function attributes in the .ll files, if you'd prefer that.

Reply:
I was thinking that might make more sense yes.   Seems like that would match better what llvm lto would produce.  I don't feel strongly though.

Reply:
Done